src/backend/app/main.py

```diff
@@ -30,7 +30,7 @@
 from osm_fieldwork.xlsforms import xlsforms_path
 
 from app.__version__ import __version__
-from app.auth import auth_routes
+from app.auth import auth_routes, temp_auth
```

Comment:
We probably don't need a new router for the single endpoint - should it go under the auth router?